.editorconfig

```diff
@@ -0,0 +1,23 @@
+root = true
+
+[*.rs]
+charset=utf-8
+end_of_line=lf
+indent_size=space
+indent_style=space
```

Comment:
Actually, we're using `Tab` instead of `Space`.

Reply:
We should use space as it doesn't conflict with Rust docstrings. Also, if you use modern editors, they respect the editorconfig file and give you the same typing experience either way.